app/assets/stylesheets/base/_values.scss

```
@@ -1 +1 @@
-$z-index-front: 9000;
+$z-index-front: 9001;
```

Could you explain why you increased this value?

---

Seconding.

---

Not a serious suggestion! I thought the original value of 9000 was cheeky, so I +1'ed it to be over 9000.  😅 
```suggestion
$z-index-front: 9000;
```
is good.